### PR TITLE
fix(GRW-1148): add support for house dk

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## What?
Tweak Edit Information Modal in order to support House DK

## Why?
So we can edit House DK quotes

Screenshot 2022-05-30 at 18 08 08

**Ticket(s): [GRW-1148](https://hedvig.atlassian.net/browse/GRW-1148)**

### CODE REVIEW

File: src/client/components/DetailsModal/Details/DanishHouseDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            name="data.householdSize"
+            field={{
+              label: 'DETAILS_MODULE_TABLE_HOUSEHOLD_SIZE_CELL_LABEL',
+              placeholder: '',
+              type: inputTypes.number,
+            }}
+            formikProps={formikProps}
+          />
+          <BooleanInput
+            name="data.isStudent"
```
Comment (simonauner) [inline src/client/components/DetailsModal/Details/DanishHouseDetails.tsx:118]: This one isn't in the yup object above?

File: src/client/components/DetailsModal/Details/DanishHouseDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      <ContentColumn>
+        <InputGroup>
+          <TextInput
+            name="data.street"
+            label="DETAILS_MODULE_TABLE_ADDRESS_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <ZipcodeInput
+            name="data.zipCode"
+            market="NO"
```
Comment (simonauner) [inline src/client/components/DetailsModal/Details/DanishHouseDetails.tsx:100]: DK?

Comment (guilhermespopolin) [reply src/client/components/DetailsModal/Details/DanishHouseDetails.tsx:100]: 🤦🏻 thanks

File: src/client/components/DetailsModal/Details/DanishHouseDetails.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply src/client/components/DetailsModal/Details/DanishHouseDetails.tsx:118]: No. This comes from formik. We'r initialising it [here](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/components/DetailsModal/index.tsx#L220)

**Review by simonauner — APPROVED** [review]
Nice one! ✅

**Review by robinandeer — APPROVED** [review]

**Pull request merged**